docs/scripts/fetch_contributors.py

```
@@ -0,0 +1,104 @@
+        return []
+    
+    human_contributors = []
+    for contributor in contributors:
+        if is_bot(contributor["login"]):
+            continue
+        
+        # Only keep necessary fields to reduce file size
+        human_contributors.append({
+            "login": contributor["login"],
+            "avatar_url": contributor["avatar_url"],
+            "html_url": contributor["html_url"]
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)